trailblazer/dto/analyses_response.py

```diff
@@ -0,0 +1,52 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+
+class Job(BaseModel):
+    analysis_id: int
+    context: str | None = None
+    elapsed: int
+    id: int
+    name: str
+    slurm_id: int
+    started_at: datetime | None = None
+    status: str
+
+
+class User(BaseModel):
+    avatar: str | None = None
```

Comment:
Is this used? Can we remove it?

Reply:
No, it is no longer used and can be removed